packages/forms/src/elements/Range.tsx

```diff
@@ -10,11 +10,16 @@ import { composeEventHandlers, useCombinedRefs } from '@zendeskgarden/container-
 import useFieldContext from '../utils/useFieldContext';
 import { StyledRangeInput } from '../styled';
 
+export interface IRangeProps extends InputHTMLAttributes<HTMLInputElement> {
```

Comment:
I'm kind of thinking I don't need this `export` since the prop is internal-only. Thoughts?

Reply:
LGTM!

Reply:
Could we use the `__NAME` scheme to denote an internal prop?